src/scandeval/model_setups/hf.py

```diff
@@ -240,94 +241,129 @@ def load_model(
-                # `pooler_hidden_size` is the same size as `hidden_size`
-                if config.model_type == "deberta-v2":
-                    config.pooler_hidden_size = config.hidden_size
-
-                with warnings.catch_warnings(), HiddenPrints():
-                    warnings.filterwarnings("ignore", category=UserWarning)
-                    warnings.filterwarnings("ignore", category=FutureWarning)
-                    try:
-                        model_or_tuple = model_cls_or_none.from_pretrained(
-                            model_config.model_id, **model_kwargs
+                    # If the model class could not be found then raise an error
+                    if not model_cls_or_none:
+                        raise InvalidBenchmark(
+                            f"The supertask {supertask!r} does not correspond to a "
+                            "Hugging Face AutoModel type (such as "
+                            "`AutoModelForSequenceClassification`)."
                         )
-                    except ImportError as e:
-                        if "flash attention" in str(e).lower():
-                            raise InvalidBenchmark(
-                                "The model you are trying to load requires Flash "
-                                "Attention. To use Flash Attention, please install "
-                                "the `flash-attn` package, which can be done by "
-                                "running `pip install -U wheel && "
-                                "FLASH_ATTENTION_SKIP_CUDA_BUILD=TRUE pip install "
-                                "flash-attn --no-build-isolation`."
+
+                    # If the model is a DeBERTaV2 model then we ensure that
+                    # `pooler_hidden_size` is the same size as `hidden_size`
+                    if config.model_type == "deberta-v2":
+                        config.pooler_hidden_size = config.hidden_size
+
+                    with warnings.catch_warnings(), HiddenPrints():
+                        warnings.filterwarnings("ignore", category=UserWarning)
+                        warnings.filterwarnings("ignore", category=FutureWarning)
+                        try:
+                            model_or_tuple = model_cls_or_none.from_pretrained(
+                                model_config.model_id, **model_kwargs
                             )
-                    except (KeyError, RuntimeError) as e:
-                        if not model_kwargs["ignore_mismatched_sizes"]:
-                            logger.debug(
-                                f"{type(e).__name__} occurred during the loading "
-                                f"of the {model_id!r} model. Retrying with "
-                                "`ignore_mismatched_sizes` set to True."
+                        except ImportError as e:
+                            if "flash attention" in str(e).lower():
+                                raise InvalidBenchmark(
```

Comment:
InvalidBenchmark, is that the right error here?

Reply:
Added a `FlashAttentionNotInstalled` exception now